scripts/rabbitmq-server-ha.ocf

```diff
@@ -66,6 +67,7 @@ OCF_RESKEY_rmq_feature_local_list_queues_default=true
 : ${OCF_RESKEY_mnesia_schema_base=${OCF_RESKEY_mnesia_schema_base_default}}
 : ${OCF_RESKEY_pid_file=${OCF_RESKEY_pid_file_default}}
 : ${OCF_RESKEY_node_port=${OCF_RESKEY_node_port_default}}
+: ${OCF_RESKEY_vhost=${OCF_RESKEY_vhost_default}}
```

Comment:
The option is renamed as suggested; I also extended the documentation for this option to stress out that it must be configured correctly.